sections/main-cart-items.liquid

```diff
@@ -116,72 +116,34 @@
                     </p>
                   </td>
 
-                  <td class="cart-item__prices right">
+                  <td class="cart-item__totals right medium-down">
```

Comment:
Unfortunately, I had to duplicate the total column because of an alignment/DOM structure issue so this is shown only in mobile